packages/eslint-plugin-x/README.md

```diff
@@ -1,11 +1,12 @@
 # ESLint Plugin X
 
-The project name is just to follow the 
```

Comment:
I know after adding the linter for `md` files this change is ok but why did linted this package?